ghascompliance/__main__.py

```diff
@@ -30,7 +30,8 @@
 parser.add_argument("--disable-dependabot", action="store_true")
 parser.add_argument("--disable-dependency-licensing", action="store_true")
 parser.add_argument("--disable-dependencies", action="store_true")
-parser.add_argument("--disable-secret-scanning", action="store_true")
+parser.add_argument("--disable-secret-scanning", action="store_true")   
+parser.add_argument("--is-github-app-token", action="store_true", default=False)
```

Comment:
@4bg0P Do you think that this argument should / could be set via the Actions?

Reply:
https://github.com/GeekMasher/advanced-security-compliance/blob/86299894aa4aedb26548f937ae281a88f6f8464e/action.yml#L4

Reply:
@GeekMasher Sure, it is a great idea to set it using Actions also!

Reply:
I'll add this as an upcoming update